tests/integration/native_extension/native_ext.c

```diff
@@ -24,7 +26,7 @@ __attribute__((noinline)) static void foo() {
 }
 
 PyObject*
-run_simple(PyObject*, PyObject*)
+run_simple(PyObject* mod , PyObject* arg)
```

Comment:
Why is this change needed? Is it actually somehow relevant to getting the test suite to pass on Mac?

Oh - does the C compiler that's being used on Mac complain if a parameter has no name?

Reply:
The c compiler in the Linux images is old enough that complains that this is invalid C